### PR TITLE
SurbhiJ_Assignment_10

### PR DESCRIPTION
**Pull request by SurbhiJOct20**
Added assignment 10

### CODE REVIEW

File: src/surbhi/ClassA.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package surbhi;
+
+public class ClassA {
+
+	void m1() {
+		System.out.println("A,m1");
+
```
Comment (KrishnaKTechnocredits) [inline src/surbhi/ClassA.java:7]: Extra blank space is not required.

File: src/surbhi/ClassA.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+		ClassB B = new ClassB();
+		B.m2();
+	}
+
+	public static void main(String[] args) {
+
```
Comment (KrishnaKTechnocredits) [inline src/surbhi/ClassA.java:13]: Remove this too

File: src/surbhi/ClassB.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package surbhi;
+
+public class ClassB {
+
+	void m2() {
+		System.out.println("m3");
+		m3();
+
```
Comment (KrishnaKTechnocredits) [inline src/surbhi/ClassB.java:8]: Remove this line too

File: src/surbhi/ClassB.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+	}
+
+	static void m3() {
+		System.out.println("m4");
+
```
Comment (KrishnaKTechnocredits) [inline src/surbhi/ClassB.java:13]: Please remove extra blank line

File: src/surbhi/ClassA.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+		ClassA A = new ClassA();
```
Comment (KrishnaKTechnocredits) [inline src/surbhi/ClassA.java:14]: reference variable should be in small letter.

File: src/surbhi/ClassB.java [hunk 6]
```diff
@@ -0,0 +1,18 @@
+package surbhi;
+
+public class ClassB {
+
+	void m2() {
+		System.out.println("m3");
```
Comment (KrishnaKTechnocredits) [inline src/surbhi/ClassB.java:6]: printing statements are wrong. you are in m2 method so why should we print "m3". I can see this mistake everywhere.

**Review by KrishnaKTechnocredits — CHANGES_REQUESTED** [review]
change the code per suggestions.

**Review by harshvegada — APPROVED** [review]
Looks all the method calling are cover so far, Good job @SurbhiJOct20

**Pull request merged**